packages/loot-core/src/server/main.ts

```diff
@@ -1492,7 +1500,7 @@ handlers['subscribe-sign-in'] = async function ({ password }) {
     return {};
   }
 
-  return { error: 'invalid-password' };
+  return { error: res.error || 'invalid-password' };
```

Comment:
```suggestion
  return { error: res.reason|| 'invalid-password' };
```
@twk3 - changing this to reason since that is what this commit now is returning: https://github.com/actualbudget/actual-server/commit/1bbba663b2c20b722327d885d0cbf80dba0ea44d#diff-024ac510e5825a4edf4be0b860f4abba35556d3ca0b5aad96c0499ed126f9452R44